modules/representation_management/app/models/representation_management/form_2122_base.rb

```diff
@@ -95,6 +100,12 @@ class Form2122Base
       validates :claimant_phone, length: { is: 10 }, format: { with: TEN_DIGIT_NUMBER }
     end
 
+    validates :representative_id, presence: true
+
+    def representative
```

Comment:
I don't think `#representative` is being called anywhere, is it? I couldn't find it being called. I pulled the branch, commented it out, ran the tests, and they all passed.

Reply:
It is used to lookup the representative values for `modules/representation_management/lib/representation_management/v0/pdf_constructor/form_2122.rb` and `modules/representation_management/lib/representation_management/v0/pdf_constructor/form_2122a.rb`.  It's used in a few tests as well.  If it's commented out `rspec modules/representation_management/spec` should get pretty ugly.